submissions/LGeorge007/DOM-practice/style.css

```diff
@@ -0,0 +1,163 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background: url("img/background.jpg");
+    background-size: cover;
+    font-family: 'Kelly Slab', cursive;
+    overflow-x: hidden;
+}
+
+article {
```

Comment:
Avoid styling by #id or HTML tag. [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, separate responsibilities:

- HTML tag for markup;
- CSS class for styling;
- Id for JS logic